chronograf/1.3/alpine/Dockerfile

```diff
@@ -6,9 +6,15 @@ RUN apk add --no-cache ca-certificates && \
 
 ENV CHRONOGRAF_VERSION 1.3.0
 
-RUN apk add --no-cache --virtual .build-deps wget gnupg tar && \
-    gpg --keyserver hkp://ha.pool.sks-keyservers.net \
-        --recv-keys 05CE15085FC09D18E99EFB22684A14CF2582E0C5 && \
+RUN set -ex && \
+    apk add --no-cache --virtual .build-deps wget gnupg tar && \
+    for key in \
+    05CE15085FC09D18E99EFB22684A14CF2582E0C5 \
+    ; do \
```

Comment:
Can you make the style here consistent between the normal and the alpine images? I prefer the current layout from the normal images to the format the alpine ones have, but consistency is key.